server/claim-review-task/claim-review-task.controller.ts

```diff
@@ -25,15 +25,14 @@ export class ClaimReviewController {
         private claimReviewTaskService: ClaimReviewTaskService,
         private captchaService: CaptchaService,
         private viewService: ViewService
-    ) { }
+    ) {}
 
     @IsPublic()
     @Get("api/claimreviewtask")
     public async getByMachineValue(@Query() getTasksDTO: GetTasksDTO) {
         const { page = 0, pageSize = 10, order = 1, value } = getTasksDTO;
         return Promise.all([
             this.claimReviewTaskService.listAll(page, pageSize, order, value),
-            // This should count the number of documents for the original query without pagination
```

Comment:
Any reason for this comment removal?

Reply:
for me, it's clear what this function does, besides that it already exists in other controllers